### ISSUE TITLE
[blocked] Detect bundled P3 application properly

### ISSUE THREAD

**Issue by web-padawan** [vaadin/vaadin-development-mode-detector#10]
needs https://github.com/vaadin/vaadin-development-mode-detector/issues/17

The current mechanism used for ES modules only takes care of the top-level scripts and is not reliable for the apps. The better option would be to use modulizer with the `--add-import-path` flag for this particular component (development mode probe), this should be supported by the CLI nowadays.

**Comment by web-padawan** [comment]
- [ ] Blocked by Polymer/polymer-bundler#624 - the bundler should not fail on the `import.meta.url` code, in case when `transformImportMeta` option passed to `polymer-build` is set to `false`

**Comment by web-padawan** [comment]
See https://github.com/Polymer/tools/issues/294#issuecomment-386778419
We will have to open a new issue to continue discussion about the topic blocking us:

> We're considering adding something like import.meta._bundleUrl to point to the actual bundled module's URL.

**Comment by web-padawan** [comment]
Closing as per #22 the "bundled" check is replaced by "minified" check
